pfunk/__main__.py

```diff
@@ -10,28 +10,31 @@
 from __future__ import print_function, unicode_literals
 
 import argparse
+import pfunk
```

Comment:
I moved these into the functions because it can take some time to import them, would rather avoid that for the command line arg where we can!

Reply:
ok, I'll get rid of this